pt-BR/mail/chrome/messenger/messenger.dtd

```diff
@@ -666,15 +666,15 @@
 <!ENTITY SendMailTo.accesskey "N">
 <!ENTITY CopyEmailAddress.label "Copiar email">
```

Comment:
@brunotiussi Understood. After receiving Sherman's approval, either use e-mail or email (chose one to keep consistency).